plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/RuntimeUtils.java

```diff
@@ -302,6 +302,19 @@ public static boolean isWindows() {
         return IS_WINDOWS;
     }
 
+
+    /**
+     * Checks if current application is shipped from Windows store
+     * @return true if shipped from Windows store, false if not.
```

Comment:
A pointless `@return` tag that quotes summary.

```suggestion
     * Checks if the current application is shipped from the Windows store
```

Reply:
I am pretty sure that our current code style settings require return for  Javadoc comments

Reply:
It doesn't.